utils/firestore.js

```diff
@@ -745,6 +745,75 @@ const removeDocumentFromCollection = async (connectID, token) => {
     }
 };
 
+/**
+ * Deletes pathology report files from Storage Bucket and file records from "pathologyReports" collection in Firestore
+ * @param {number} connectId
+ * @returns Promise<void>
+ */
+const deletePathologyReports = async (connectId) => {
+  const tierStr = process.env.GCLOUD_PROJECT?.split("-").slice(3, 5).join("-").toLowerCase() || "";
```

Comment:
might be easier / cleaner to just create an environment variable in the config yaml file?

Reply:
The `GCLOUD_PROJECT` defined inside `.env.yaml` is good enough. We may not need to create environment variables for special cases like this.
For bucket names, leads suggested to use shorter (`connect-dev` for dev tier) instead of longer string (`nih-nci-dceg-connect-dev` or `dceg-connect-dev` for dev tier). Thus the truncated project name is used here to find buckets.